sagemaker-python-sdk/dgl_gcmc/mxnet_gcmc_hypertune.ipynb

```diff
@@ -241,6 +241,7 @@
     "    tags=task_tags,\n",
     "    max_jobs=10,\n",
     "    max_parallel_jobs=2,\n",
+    "    base_tuning_job_name=\"mxnet-gcmc-tune\"\n",
```

Comment:
Does specifying a base tuning job name automatically append the datetime to each tuning job created when this notebook is re-run?

Reply:
Yes, it does - generally a "base name" is the prefix while a date is automatically appended. Whereas a "name" is the full name and I will have to add a date manually to it.

Reply:
The full name that appeared in my testing of this notebook is `mxnet-gcmc-tune-220426-2021`